src/react-native/11-navigation/navigation.md

```diff
@@ -64,11 +161,73 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 
 ## Objective 2: Using React Native stack navigation
 
-TODO
-
-### Concept TODO
-
-TODO
+Now that we've covered tab navigation, let's move on to stack navigation. As mentioned in the previous section, most applications use a combination of both tab and stack navigation to create a seamless user experience.
+
+### Stack vs Native Stack
+
+The React Navigation library provides both a stack navigator as well as a native stack navigator. The native stack navigator uses the native APIs provided by the platform, `UINavigationController` on iOS and `Fragment` on Android. Because it is using the native APIs, it provides native performance and exposes native-specific features. Although the stack navigator is not as performant, it is an easier-to-use, more customizable, JavaScript-based alternative.
+
+### Performance
+
+Both the stack navigator and tab navigator have an optional prop, `detachInactiveScreens`, that defaults to `true`. This boolean value indicates whether inactive screens (screens that are not currently visible) should be detached from the view hierarchy. This can improve performance by reducing the number of views that need to be rendered and maintained in memory.
```

Comment:
I think this is interesting, and we will probably skip explaining this during the training.

Reply:
👍 i will leave it in the academy for now, but skip in slides?